closure/goog/net/xhrio.js

```diff
@@ -530,8 +539,9 @@ goog.net.XhrIo.prototype.send = function(
 
   // Set up upload/download progress events, if progress events are supported.
   if (this.getProgressEventsEnabled() && 'onprogress' in this.xhr_) {
-    this.xhr_.onprogress =
-        goog.bind(function(e) { this.onProgressHandler_(e, true); }, this);
+    this.xhr_.onprogress = goog.bind(function(e) {
```

Comment:
@code-4-fun Ohh right. The closure-library seems to still use [3.7.1](https://github.com/google/closure-library/blob/8b3e2795d3b2d40d67eacc2bfa02c30718eda04c/scripts/ci/install_closure_deps.sh#L6) at the moment.
@joeltine can you bump up the clang format version?
Note: The change was included in the clang release 3.8.0 but I do have trouble finding the commit that introduced this change in clang-format. (Hopefully this change was intentional.)